Add issue templates (#935)
TypeStrong/typedoc: .github/ISSUE_TEMPLATE/suggestion.md

---

> name: Suggestion

Author: ```suggestion
name: Feature Request
```

> about: Suggest an improvement or a feature

Reviewer: Could we rename this template "Feature Request"?

Author: @aciccarello actually I was hesitating for minutes on that issue. For me, "Feature Request" sounds like... a narrower name. For example, #898, #935, #917 could be suggestions, but not feature requests. Feature requests sound like only an improvement only in typedoc code to me. Is that the only case for me?

If you get what I mean, "Suggestion" sounds like a superset of "Feature Request", which might embrace more issues.

What are your thoughts?

Reviewer: That's a good point. I still lean towards "Feature Request" since it is a more standard term and fits the contents of the template better. We could possibly add a more general option in the future.

Author: Yeap!

---
<!--
    Thank you for contributing to Typedoc! 
    
    First, please use a clear and descriptive title for the issue to identify the suggestion.
    

>     Please be as specific as possible! Provide codes, sreenshots, repositories, explanations, issues, links ... anything that could describe your point.

Reviewer: Minor typo
```suggestion
    Please be as specific as possible! Provide codes, screenshots, repositories, explanations, issues, links ... anything that could describe your point.
```

-->

- [ ] I have checked [issues with enhancement label](https://github.com/TypeStrong/typedoc/labels/enhancement) and found no duplicates

## Problem
<!--

>     What is making you feel inconvenient without this improvement or feature?

Reviewer: ```suggestion
    What is missing or inconvenient without this improvement or feature?
```

-->

## Suggestion
<!--

>     What do you suggest to fix the problem?

Reviewer: ```suggestion
    How do you suggest fixing the problem?
```
